crates/symbolicator/src/services/cacher.rs

```diff
@@ -477,6 +462,29 @@ impl<T: CacheItemRequest> Cacher<T> {
             } else {
                 // A concurrent cache lookup is considered new. This does not imply a cache miss.
                 metric!(counter(&format!("caches.{}.channel.miss", name)) += 1);
+
+                // We count down towards zero, and if we reach or surpass it, we will short circuit here.
+                // Doing the short-circuiting here means we don't create a channel at all, and don't
+                // put it into `current_computations`.
+                // Previously, that could have potentially returned the error to a non-lazy
+                // computation requests based on the `key`.
+                let max_lazy_refreshes = self.config.max_lazy_refreshes();
+                if is_refresh && max_lazy_refreshes.fetch_sub(1, Ordering::Relaxed) <= 0 {
+                    max_lazy_refreshes.fetch_add(1, Ordering::Relaxed);
+
+                    metric!(counter(&format!("caches.{}.lazy_limit_hit", name)) += 1);
```

Comment:
can we put the cache name in a tag instead of interpolating the metric name?  I know there's other metrics that do this, but let's not do this for new metrics at least.

Reply:
fair enough, I think changing all the others might involve more work also in DD, etc.

Reply:
yes, no need to change existing metric names